Makefile

```diff
@@ -26,23 +26,31 @@
 
 help::
 	@echo ' '
-	@echo 'Usage:'
+	@echo 'Usage: make <target>'
 	@echo ' '
-	@echo 'To build all sample programs and run all unit-tests:'
+	@echo 'Supported targets:'
+	@echo '    all-c-tests all-cpp-tests all-cc-tests all-unit-tests all'
+	@echo '    run-c-tests run-cpp-tests run-cc-tests run-unit-tests run-tests'
+	@echo '    clean'
+	@echo ' '
+	@echo 'To build all sample programs and run unit-tests:'
+	@echo ' make clean && CC=g++ CXX=g++ LD=g++ make run-unit-tests'
+	@echo ' '
+	@echo 'To build all sample programs and run all tests:'
 	@echo ' make clean && CC=g++ CXX=g++ LD=g++ make all'
 	@echo ' make run-tests'
 	@echo ' '
-	@echo 'To build C-sample programs and run unit-tests:'
+	@echo 'To build C-sample programs and run tests:'
```

Comment:
Edited these messages to say, simply, `run tests` rather than `run unit-tests` as the latter term is now reserved for the newly added unit-test.